tests/scripts/thread-cert/Cert_7_1_06_BorderRouterAsLeader.py

```diff
@@ -255,22 +256,20 @@ def verify(self, pv):
                                   LEADER_DATA_TLV
                                  } <= set(p.mle.tlv.type) and\
                                  {
-                                  NWD_BORDER_ROUTER_TLV,
                                   NWD_BORDER_ROUTER_TLV,
                                   NWD_6LOWPAN_ID_TLV
                                  } <= set(p.thread_nwd.tlv.type) and\
-                       p.thread_nwd.tlv.border_router.flag.p == [1, 1] and\
-                       p.thread_nwd.tlv.border_router.flag.s == [1, 1] and\
-                       p.thread_nwd.tlv.border_router.flag.r == [1, 1] and\
-                       p.thread_nwd.tlv.border_router.flag.o == [1, 1] and\
                        p.mle.tlv.leader_data.data_version ==
                        (_dr_pkt.mle.tlv.leader_data.data_version + 1) % 256 and\
-                       (p.mle.tlv.leader_data.stable_data_version ==
-                        (_dr_pkt.mle.tlv.leader_data.stable_data_version + 1) % 256 or\
+                       ((p.mle.tlv.leader_data.stable_data_version ==
+                       (_dr_pkt.mle.tlv.leader_data.stable_data_version + 1) % 256 or\
                        p.mle.tlv.leader_data.stable_data_version ==
-                        (_pkt.mle.tlv.leader_data.stable_data_version + 1) % 256) and\
-                       [Ipv6Addr(PREFIX_2001[:-3])] ==
-                       p.thread_nwd.tlv.prefix
+                       (_pkt.mle.tlv.leader_data.stable_data_version + 1) % 256)) and\
+                       is_sublist([1, 1], p.thread_nwd.tlv.border_router.flag.p) and\
+                       is_sublist([1, 1], p.thread_nwd.tlv.border_router.flag.s) and\
+                       is_sublist([1, 1], p.thread_nwd.tlv.border_router.flag.r) and\
+                       is_sublist([1, 1], p.thread_nwd.tlv.border_router.flag.o) and\
+                       is_sublist([Ipv6Addr(PREFIX_2001[:-3])], p.thread_nwd.tlv.prefix)
```

Comment:
also apply `is_sublist` to above
https://github.com/openthread/openthread/blob/5cbd4be2d9515fe24e179c238a9c7c055b8eeb58/tests/scripts/thread-cert/Cert_7_1_06_BorderRouterAsLeader.py#L245

and below
https://github.com/openthread/openthread/blob/5cbd4be2d9515fe24e179c238a9c7c055b8eeb58/tests/scripts/thread-cert/Cert_7_1_06_BorderRouterAsLeader.py#L353